### PR TITLE
Made title more human-readable

### PR DESCRIPTION
**Pull request by nlharris**
and other minor tweaks to wording

### CODE REVIEW

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
Looks great! Am I allowed to merge? Or you should go ahead.

**Comment by nlharris** [comment]
I don't know what the official policy is--whether the approver can merge or whether the original creator of the PR should merge after getting approval. I'll go ahead and merge this one.

**Pull request merged**
